addons/web_editor/static/lib/odoo-editor/test/spec/color.test.js

```diff
@@ -1,16 +1,16 @@
 import { BasicEditor, testEditor } from '../utils.js';
 
-const setColor = (color, mode) => {
-    return async editor => {
-        await editor.execCommand('applyColor', color, mode);
+const setColor = (color, [...modes]) => {
+    return editor => {
+        modes.forEach(mode => editor.execCommand('applyColor', color, mode));
```

Comment:
IMO there is no need for that. In the few places where you need to set two modes, just call `setColor` twice in a callback.